WowPacketParser/Loading/BinaryPacketReader.cs

```diff
@@ -207,6 +207,7 @@ public Packet Read(int number, string fileName)
                             cIndex = _reader.ReadInt32(); // session id, connection index
                             var tickCount = _reader.ReadUInt32();
                             time = _startTime.AddMilliseconds(tickCount - _startTickCount);
+                            time = TimeZone.CurrentTimeZone.ToLocalTime(time);
```

Comment:
Perhaps `time = TimeZone.CurrentTimeZone.ToLocalTime(_startTime.AddMilliseconds(tickCount - _startTickCount));` ? I don't really like changing values of existing values.